src/libraries/System.IO.FileSystem/src/System/IO/DirectoryInfo.cs

```diff
@@ -235,8 +241,8 @@ public void MoveTo(string destDirName)
             Invalidate();
         }
 
-        public override void Delete() => FileSystem.RemoveDirectory(FullPath, recursive: false);
+        public override void Delete() => DeleteAndInvalidate(false);
```

Comment:
```suggestion
        public override void Delete() => Delete(false);
```

You do not really need a new method for this.

Reply:
Agreed. Changed